src/core/matcher.cpp

```diff
@@ -71,10 +71,16 @@ class wf::view_matcher_t::impl
         }
     }
 
+    impl() = default;
     ~impl()
     {
         disconnect_updated_handler();
     }
+
+    impl(const impl &) = default;
+    impl(impl &&) = default;
```

Comment:
Is here the default that the constructor is deleted? If yes, i'd rather have an explicit `=delete`

Reply:
No this default isn't "implicitly deleted" by anything.